doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md

```diff
@@ -336,12 +338,12 @@ And we also need two corresponding test classes:
     ```java
     @Override
     protected RemoteMetadataReader createRemoteMetadataReader() {
-        return new BaseRemoteMetadataReader(this.connection, this.properties);
+        return new BaseRemoteMetadataReader(this.connectionFactory.getConnection(), this.properties);
```

Comment:
I am guessing there is a `try { ... } catch { ... }` block missing here.

Reply:
done